### ISSUE TITLE
javalib 2.3.5 doesn't want to be sandboxed

### ISSUE THREAD

**Issue by jvillard** [ocaml/opam-repository#13040]
I cannot install javalib with sandboxing enabled:

```
#=== ERROR while compiling javalib.2.3.5 ======================================#
# context     2.0.1 | linux/x86_64 | ocaml-variants.4.07.1+flambda | https://opam.ocaml.org/#0f382fae
# path        ~/.opam/ocaml-variants.4.07.1+flambda/.opam-switch/build/javalib.2.3.5
# command     ~/.opam/opam-init/hooks/sandbox.sh build make installptrees
# exit-code   2
# env-file    ~/.opam/log/javalib-3648-a71bb4.env
# output-file ~/.opam/log/javalib-3648-a71bb4.out
### output ###
# make -C ptrees install
# make[1]: Entering directory '/home/jul/.opam.1.2.2/ocaml-variants.4.07.1+flambda/.opam-switch/build/javalib.2.3.5/ptrees'
# /home/jul/.opam/ocaml-variants.4.07.1+flambda/bin/ocamlfind install ptrees META ptrees.cma ptrees.cmxa ptmap.mli ptset.mli genericSet.mli genericMap.mli ptmap.cmi ptset.cmi genericSet.cmi genericMap.cmi ptrees.a
# ocamlfind: Cannot mkdir /home/jul/.opam/ocaml-variants.4.07.1+flambda/lib/ptrees: Read-only file system
# make[1]: *** [Makefile:30: install] Error 2
# make[1]: Leaving directory '/home/jul/.opam.1.2.2/ocaml-variants.4.07.1+flambda/.opam-switch/build/javalib.2.3.5/ptrees'
# make: *** [Makefile:14: installptrees] Error 2
```

Looks like the "build" step of javalib tries to install "ptrees" into the current opam switch, which I'm guessing the sandbox doesn't like:

https://github.com/ocaml/opam-repository/blob/0f382fae9724fc81c50f511a28efa1609505c613/packages/javalib/javalib.2.3.5/opam#L7-L12

Disabling the sandbox allows me to install javalib.

**Comment by rjbou** [comment]
Related topic [discuss](https://discuss.ocaml.org/t/install-opam-1-package-via-opam-2/2776) & [javalib](https://gforge.inria.fr/tracker/index.php?func=detail&aid=21722&group_id=686&atid=2815)

**Comment by jvillard** [comment]
Resolved by #13056.
